source/util/FileUtils.cpp

```diff
@@ -302,3 +302,26 @@ bool FileUtils::FileExists(const string &filename)
     ifstream f(expandedPath);
     return f.good();
 }
+
+bool FileUtils::IsValidFilePath(const string &filePath)
+{
+    wordexp_t word;
+    switch (wordexp(filePath.c_str(), &word, 0))
```

Comment:
I think at this line of code we will have segmentation fault if the file path is incorrect.